### PR TITLE
feat: Quarkus port is inferred from application.properties/yaml (cons…

### PR DESCRIPTION
**Pull request by manusa**
## Description
feat: Quarkus port is inferred from application.properties/yaml (considers profile too)

## Type of change

 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [x] Feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change
 - [ ] Chore (non-breaking change which doesn't affect codebase;
   test, version modification, documentation, etc.)

## Checklist
 - [x] I have read the [contributing guidelines](https://www.eclipse.org/jkube/contributing)
 - [x] I signed-off my commit with a user that has signed the [Eclipse Contributor Agreement](https://www.eclipse.org/legal/ECA.php)
 - [x] I Added [CHANGELOG](../CHANGELOG.md) entry
 - [x] I have implemented unit tests to cover my changes
 - [ ] I have updated the [documentation](../kubernetes-maven-plugin/doc) accordingly
 - [x] No new bugs, code smells, etc. in [SonarCloud](https://sonarcloud.io/dashboard?id=jkubeio_jkube) report
 - [x] I tested my code in Kubernetes
 - [ ] I tested my code in OpenShift

### CODE REVIEW

File: jkube-kit/jkube-kit-quarkus/src/test/java/org/eclipse/jkube/quarkus/generator/QuarkusGeneratorTest.java [hunk 1]
```diff
@@ -33,6 +33,7 @@
 
 import mockit.Expectations;
 import mockit.Mocked;
+import org.eclipse.jkube.quarkus.QuarkusUtilsTest;
```
Comment (rohanKanojia) [inline jkube-kit/jkube-kit-quarkus/src/test/java/org/eclipse/jkube/quarkus/generator/QuarkusGeneratorTest.java:36]: unused import

File: jkube-kit/jkube-kit-quarkus/src/test/java/org/eclipse/jkube/quarkus/generator/QuarkusGeneratorExtractPortsTest.java [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+package org.eclipse.jkube.quarkus.generator;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Collections;
```
Comment (rohanKanojia) [inline jkube-kit/jkube-kit-quarkus/src/test/java/org/eclipse/jkube/quarkus/generator/QuarkusGeneratorExtractPortsTest.java:19]: unused import

**Review by rohanKanojia — APPROVED** [review]

**Comment by manusa** [comment]
This is just a test, please ignore

**test**

**Comment by manusa** [comment]
This is just a test, please ignore

**test**

**Comment by manusa** [comment]
This is just a test, please ignore

**test**

**Comment by manusa** [comment]
This is a comment.

  **Hello**

  Please ignore

**Comment by manusa** [comment]
This is a comment.

  **Hello**

  Please ignore

**Pull request merged**